gammapy/cube/core.py

```diff
@@ -526,6 +526,14 @@ def to_fits(self):
 
         return hdu_list
 
+
+    def to_image_list(self):
+        """ Writes SkyCube to `gammapy.image.SkyImageList` as a list of `gammapy.image.SkyMap` objects.
+        """
+        skymaps = [SkyMap(self.name, Quantity(slice_data, self.data.unit), self.wcs) for slice_data in self.data]
```

Comment:
Good point!

I printed the wcs that I got from FermiGalacticCenter.diffuse_model() and got

```
WCS Keywords
Number of WCS axes: 2
CTYPE : 'GLON-CAR'  'GLAT-CAR'  
CRVAL : 0.0  0.0  
CRPIX : 31.5  11.5  
PC1_1 PC1_2  : 1.0  0.0  
PC2_1 PC2_2  : 0.0  1.0  
CDELT : 0.5  0.5  
NAXIS    : 61 21

```
Is this an error in `FermiGalacticCenter.diffuse_model()` data files, should I look into this? The code is 

```

    @staticmethod
    def diffuse_model():
        """Diffuse model (`~gammapy.data.SkyCube`)"""
        from ..cube import SkyCube
        filename = FermiGalacticCenter.filenames()['diffuse_model']
        return SkyCube.read(filename)
```

So as far as I understood from tutorial, the SkyCube usually uses celestial wcs -- longtitute, latitude and cubeface. Can I rely that cubeface is the first axis or should I check that explicitly in `cube.sky_image(slice_pos)`?